Vagrantfile

```diff
@@ -6,72 +6,21 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+bash ./scripts/install_prerequisites.sh
+bash ./scripts/start.sh
+SCRIPT
 
 Vagrant.configure(2) do |config|
-  # The most common configuration options are documented and commented below.
-  # For a complete reference, please see the online documentation at
-  # https://docs.vagrantup.com.
-  
-  config.vm.provider "virtualbox" do |v|
-    v.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
+    config.vm.provider "virtualbox" do |v|
+      v.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
+      v.memory = 1024
     end
-
-  # Every Vagrant development environment requires a box. You can search for
-  # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "hashicorp/precise32"
-
-  # Disable automatic box update checking. If you disable this, then
-  # boxes will only be checked for updates when the user runs
-  # `vagrant box outdated`. This is not recommended.
-  # config.vm.box_check_update = false
-
-  # Create a forwarded port mapping which allows access to a specific port
-  # within the machine from a port on the host machine. In the example below,
-  # accessing "localhost:8080" will access port 80 on the guest machine.
+  config.vm.provision "shell", inline: 'echo "export VAGRANT=true" >> ~/.profile'
```

Comment:
Could you add a comment here that this is necessary for scripts/install_prerequisites.sh to do the right thing?